java-checks-test-sources/default/pom.xml

```diff
@@ -502,7 +502,7 @@
     <dependency>
       <groupId>org.apache.commons</groupId>
       <artifactId>commons-lang3</artifactId>
-      <version>3.12.0</version>
+      <version>3.17.0</version>
```

Comment:
Fyi `secure()` is available since 3.16.0 (https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/RandomStringUtils.html#secure())